### PR TITLE
CE-2574 Adding files for Sphinx

### PR DESCRIPTION
**Pull request by cr-jeremybrown**
Adding the necessary files for Sphinx to build successfully

### CODE REVIEW

**Comment by cr-jeremybrown** [comment]
@cr-markhoeber  @cr-vickilindem  I was able to successfully build this project after adding the files in this commit. However - let me know what is missing here so we can update it as needed.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	@echo "Nest HTML files generated."
+	rm -rf $(BUILDDIR)/html/_sources
+	rm -rf $(BUILDDIR)/html/_raw_sources
+	rm -rf $(BUILDDIR)/html/search.html
+	rm -rf $(BUILDDIR)/html/searchindex.js
+	rm -rf $(BUILDDIR)/html/.buildinfo
+	rm -rf $(BUILDDIR)/html/objects.inv
+	@echo
+	@echo "Unnecessary files removed."
+	cd $(BUILDDIR) && zip -r EN_api_documentation_all.zip html \
```
Comment (cr-markhoeber) [inline Makefile]: make sure to change file name that's created

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	$(SPHINXBUILD) -D language=ja -b html $(ALLSPHINXOPTS) $(JA_BUILDDIR)/html
+	echo 
+	@echo "JA Nest HTML files generated."
+	rm -rf $(JA_BUILDDIR)/html/_sources
+	rm -rf $(JA_BUILDDIR)/html/_raw_sources
+	rm -rf $(JA_BUILDDIR)/html/search.html
+	rm -rf $(JA_BUILDDIR)/html/searchindex.js
+	@echo
+	@echo "Unnecessary files removed."
+	cd $(JA_BUILDDIR) && zip -r JA_api_documentation_all.zip html \
```
Comment (cr-markhoeber) [inline Makefile]: same on file names

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,75 @@
+
```
Comment (cr-markhoeber) [inline .gitignore]: I think we can just make git ignore:  build/*

**Comment by cr-vickilindem** [comment]
@cr-jeremybrown Looks good to me. I can't remember if the sphinx images extension was giving us issues or not, but if it's running then it's good!

**Pull request merged**